### ISSUE TITLE
Please update the release build with latest TWL_FIRM patches....

### ISSUE THREAD

**Issue by ApacheThunder** [roxas75/rxTools#462]
This is more of a suggestion. But also an issue. The 3.0 Beta2 doesn't have the last TWL_FIRM patch that allows NULL sigs. As a result I'm getting a lot of users complaining about the "Error has occurred" problem when they try various custom DSiWare SRLs. I keep having to tell them to not use 3.0 Beta2 and find a nightly build that is newer.

I suggest you either pull the build or at least add a beta3 of that build with the updated TWL elf file. It's causing confusion and I would like to see it rectified.

**Comment by mariogamer2** [comment]
This must be good (with up-to-date cakebrah).

http://www.multiupfile.com/f/ed84f196

**Comment by ApacheThunder** [comment]
Yeah there is currently a divide between users who have prepatched CIA of TWL_FIRM (I am part of this group) and those who use rxTools. The CIAs have long been updated with the new patch. But rxTools still doesn't have the main release page updated. This is the main source of the issue here. :P

**Comment by mariogamer2** [comment]
I know I just upload it to simplify the add :P
